src/vss_tools/exporters/binary.py

```diff
@@ -140,9 +146,17 @@ def cli(
         uuid=uuid,
         quantities=quantities,
         units=units,
+        types=types,
         overlays=overlays,
     )
+
     log.info("Generating binary output...")
+    if datatype_tree:
+        if types_output:
```

Comment:
What happens if types_output is not defined?
Usually other exporters export the types then bundled into the main file.
I think either it needs to throw an error in that case for the user or defining a default types output file or ...

Reply:
Test shows that if --types-output is not used in the make command the second if statement becomes false.

Reply:
does not really answer my question because `if datatype_tree` is still true then but we are not doing anything with that info in that case

Reply:
Ulf - I suggest you add an else statement, something like

```
else:
  log.warning("To generate datatype tree output you must specify wanted output file")
```
or 

```
else:
  log.warning("Ignoring type tree generation as no file name specified")
```


... that gives a hint that they maybe have forgotten to specify output file.